### PR TITLE
Import svg and collada via threejs geometry

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/threejs/readCollada.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {
+  fromColladaToThreejs,
+  fromThreejsToGeometry,
+} from '@jsxcad/convert-threejs';
+
+import { Shape } from '@jsxcad/api-shape';
+import { read } from '@jsxcad/sys';
+
+export const readCollada = async (path, { src, invert = false } = {}) => {
```
Comment (BarbourSmith) [inline api/threejs/readCollada.js:9]: It's pretty sweet how compact these become once the piping is in

File: convert/threejs/fromColladaToThreejs.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  const threejs = await fromColladaToThreejs(data);
+  const geometry = JSON.parse(
+    JSON.stringify(
+      prepareForSerialization(await fromThreejsToGeometry(threejs))
+    )
+  );
+  const expectedGeometry = JSON.parse(
+    readFileSync('duck_triangles.json', 'utf8')
+  );
+  t.deepEqual(geometry, expectedGeometry);
```
Comment (BarbourSmith) [inline convert/threejs/fromColladaToThreejs.test.js:25]: Sometimes I forget how good the testing setup is. I think that was a critical decision getting the testing in early and maintaining it. We used to have regressions all the time, but they haven't been a problem in a while.

**Review by BarbourSmith — APPROVED** [review]
Looks great to me.


Out of curiosity...If we use three for STL generation will the resulting .STL file be any different? Or is the conversion from cgal geometry to stl geometry 1:1 enough that they should be identical?

**Comment by pentacular** [comment]
> 

You might find a different arrangement of triangles, but I wouldn't expect it to matter much -- STL is pretty trivial.

**Comment by BarbourSmith** [comment]
That's kind of what I was guessing

**Pull request merged**